content/docs/user-guide/how-to/undo-adding-data.md

```diff
@@ -0,0 +1,46 @@
+# Undo Adding Data
+
+There are situations where you `dvc add` a data file and want DVC to stop
+tracking that file. Follow the steps listed here to undo `dvc add`.
+
+Let's first add a data file into an example <abbr>project</abbr> using
+`dvc add`, which creates a `.dvc` file to track the data:
+
+```dvc
+$ dvc add data.csv
+$ ls
+data.csv    data.csv.dvc
+```
+
+Now let's reverse this action:
```

Comment:
Question: how does this play with [git revert](https://git-scm.com/docs/git-revert) (if at all?) Do we need some sort of warning somewhere? Maybe not but worth considering. WDYT @imhardikj?

Reply:
Using `dvc remove` will modify `.gitignore`  so we'll have to stash or commit this change before git revert. 
Also using `dvc checkout` or `dvc pull` will after following this procedure will delete data file as `.dvc` file doesn't exist anymore. I think it would we good idea to track data file by git after removing from dvc to avoid any data loss.

Reply:
I was suggesting to use `git revert` instead of the manual procedure. Have you tried that? Could it work? Under which conditions? Should we recommend it?

> using dvc checkout or dvc pull will after following this procedure will delete data file

Not sure what you mean or how that is realted. `dvc remove` already does that.

> good idea to track data file by git 

That's not a good idea 🙂 The whole point tracking data with DVC is that you don't want to put it in Git. Maybe I'm not following your line of thought, can you explain what you mean again? Thanks

Reply:
Sorry I got confused. I tried `git revert`:
```git
$ git revert commit
On branch master
Changes to be committed:
	modified:   .gitignore
	deleted:    data.csv.dvc
Untracked files:
	data.csv
```
If we revert the commit in which we added the data file, it will delete `.dvc` file and modify `.gitignore`. But I think it's not a good idea since it will also revert other changes that were done in that commit. 
> Do we need some sort of warning somewhere? 

I don't think it's required. Users might know that revert will undo all the changes of that commit.

Reply:
> it will also revert other changes that were done in that commit

True, I guess it's only useful in very limited situations so not worth recommending.

Alright, thanks. Moving on!